src/vm/memory/memory.zig

```diff
@@ -466,3 +526,35 @@ test "Memory: getRelocatable should return ExpectedRelocatable error if Felt ins
         memory.getRelocatable(Relocatable.new(10, 30)),
     );
 }
+
+test "Memory: Memory Cell getValue" {
+    // Test setup
+    var memory = try Memory.init(std.testing.allocator);
+    defer memory.deinit();
+
+    var cell = MemoryCell.new(MaybeRelocatable{ .relocatable = Relocatable.new(1, 0) });
+
+    // Test checks
+    try expectEqual(
+        cell.getValue(),
+        MaybeRelocatable{ .relocatable = Relocatable.new(1, 0) },
```

Comment:
In Zig, I know this is confusing, but the order is inversed in the `expectEqual`: expected value is in first position :)